src/main/java/co/cask/cdc/plugins/common/References.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
```

Comment:
update copyright year

Reply:
removed this class and avoided duplication of code from IdUtils